sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -339,9 +336,9 @@ public:
   node add(const property_list& propList = {});
 
   template<typename T>
-  node add(T cgf, const property_list& propList = {});
+  node add(const device& syclDevice, T cgf, const property_list& propList = {});
 
-  node add_malloc_device(void*& data, size_t numBytes, const property_list& propList = {});
+  node add_malloc_device(const device& syclDevice, void*& data, size_t numBytes, const property_list& propList = {});
```

Comment:
Added a device parameter here for consistency, although it's likely this entry-point could be deleted